src/app/config.rs

```diff
@@ -158,6 +158,7 @@ pub trait Strings {
 /// An implementation of `Strings` that reads the values from environment variables.
 pub struct Env;
 
+#[derive(Clone)]
```

Comment:
just curious.... can this be `#[cfg(test)]`?

Reply:
I don't think so, since the tests are compiled as the binary in test mode with the proxy as just a regular dependency.